docs/proposals/compute-resource-metrics-api.md

```diff
@@ -132,16 +122,12 @@ As well as the common query parameters:
 
```

Comment:
I think maybe we should remove the field selector. I'm not sure it makes sense in this context.